m2stats/percentiles/percentilesforFLUID.pro

```diff
@@ -0,0 +1,2197 @@
+PRO fluid_git
```

Comment:
I don't really know how IDL works...but assuming it doesn't break anything, it's probably a good idea to synchronize the name of the file with the name of the program.

```suggestion
PRO percentilesforFLUID
```